src/Config.cpp

```diff
@@ -461,6 +461,7 @@ compiler_type_to_string(CompilerType compiler_type)
 
     CASE(clang);
     CASE(gcc);
+    CASE(icl);
```

Comment:
Please add `icl` to [Config::parse_compiler_type](https://github.com/ccache/ccache/blob/9536b4fb1f653fb4e10dd84f1569f5d5d732795f/src/Config.cpp#L240) and [the manual](https://github.com/ccache/ccache/blob/9536b4fb1f653fb4e10dd84f1569f5d5d732795f/doc/MANUAL.adoc#L516-L532) as well.

Reply:
Fixed